### ISSUE TITLE
Error on translate

### ISSUE THREAD

**Issue by vyper** [guru-pr/gurupr#70]
![screen shot 2015-07-02 at 11 53 45 pm](https://cloud.githubusercontent.com/assets/22549/8491795/a4606938-2115-11e5-8a94-6c0af63f9b0b.png)

**Comment by adilsoncarvalho** [comment]
The _problem_ is [here](https://github.com/guru-pr/gurupr/blob/master/app/controllers/application_controller.rb#L33-L37) as it isn't prepared to handle such situations where it's needed to supply values to do the interpolation on the locale string.

```ruby
def title
  title = @title || t('.title')
  "#{title} - #{t('layouts.application.title_suffix')}"
end
```

We got two ways to handle it for me:
- allow views to override `title` to provide whatever they want
- allow `title` to be able to reach the values it needs

I prefer the first option as it seems more clear.

**Comment by lccezinha** [comment]
What about using [this](https://github.com/fnando/page_title) to solve page_title problem ?

**Comment by adilsoncarvalho** [comment]
It seems interesting @lccezinha

**Comment by lccezinha** [comment]
I never used, but seems a god option what do you thing @vyper ?

**Comment by vyper** [comment]
Maybe... But I think that is a good choice to try!

Can you do a pull request @lccezinha? (;

**Comment by lccezinha** [comment]
@vyper I wish, but in this moment I'm very busy about my TCC, so I don't have much time NOW. In the next weeks I'll have more free time and can do a pull request about it.

**Comment by RafaelChefe** [comment]
Hi! New guy on the block, Vyper knows me from old times, and I know a few of the members...

About this issue:

I tried the page_title gem, and in order to make it work it would be necessary to:
- add the required i18n scopes
- change the structures of all the titles there are already working
- and even so, it would be needed to add something like `page_title[:name] = @workshop.name` in every controller, to get the name of the record being edited.

So I tried a different approach: added a bit of code which tries to get the current record based on the controller name. If it finds it, it passes the name to the title, if not, passes a nil options hash, which is harmless.

I'm opening a pull request now for your appreciation. =)
